homeassistant/components/homekit/accessories.py

```diff
@@ -137,6 +138,24 @@ def update_state(self, new_state):
         """
         raise NotImplementedError()
 
+    def call_service(self, acc_domain, service, params, logbook_message=None):
+        """Sent event for logbook that a change was initiated from homekit.
+
+        Then call the service to change the state.
+        This method is to be called from methods that change states using
+        self.call_service(DOMAIN, service, params, logbook_message).
+        """
+        if logbook_message is None:
+            logbook_message = service
+
+        self.hass.bus.fire(EVENT_HOMEKIT_CHANGED, {
+            ATTR_ENTITY_ID: self.entity_id,
+            'display_name': self.display_name,
```

Comment:
Maybe a new constant: `ATTR_DISPLAY_NAME`? We would need `ATTR_VALUE` as well. A new section between `Misc` and `Config` named `Attributes` would be optimal.

Reply:
These would go into the HomeKit constants then?

Reply:
:+1:

Reply:
Added const.py for HomeKit.